content/zh/case-studies/buffer/index.html

```diff
@@ -0,0 +1,157 @@
+---
+title: Buffer Case Study
```

Comment:
`Buffer Case Study` is untranslated. Is that right?

Reply:
Yes,I was negligent.
Thanks!